devhelp/data/Makefile.am

```diff
@@ -0,0 +1,10 @@
+include $(top_srcdir)/build/vars.docs.mk
+
+plugin = devhelp
+
+dist_plugindata_DATA = \
+	devhelp-plugin.svg \
+	devhelp-plugin-48.png \
+	geany-devhelp-plugin.png \
```

Comment:
`geany-devhelp.png` is missing here or it is not used and can be removed?

Reply:
Done.

Reply:
The file is still not referenced here or am I missing something?

`dist_plugindata_DATA` ensures that files listed here will be included in releases and get installed properly. If you do not add the image filename here, it won't be installed.